image/templates/helm/stackrox-central/internal/config-shape.yaml

```diff
@@ -41,11 +41,6 @@ central:
     name: null # string
     tag: null # string
     fullRef: null # string
-  dbImage:
```

Comment:
I think that it's cleaner at the top level to not embed different sets of values for different services. I would suggest that for each service/deployment that we keep the configuration separate from the other services. so that each service has it's own resources field as opposed to dbResources field. Where this does get a bit hairy is when talking about the connection string that links Central <-> Central DB. I opted for having this defined in centralDB but perhaps that is not the right place

Reply:
I will actually move this underneath central as a db: setting